May Bugfix Patch Day

BiccyThiccy opened the pull request:
[Plz put fixes in the patch notes as you make them](https://github.com/deliciousmods/1956_beta/wiki/May-2025-Patch-Notes)

BiccyThiccy committed:
Many ROM Spelling Fixes

katsuragiyy committed:
Albania stuffs

BiccyThiccy committed:
Amending previous commit + small ALB polish

Commit:
Small fix

Commit:
removed trait duplication for TUR

Commit:
quick fix

Commit:
error.log cleanup

katsuragiyy committed:
Tahiti Mangoes Icon fix

katsuragiyy committed:
EU-Micronation shenanigans

Just added generic_skip_line_tt thing to Liechtenstein and co.

katsuragiyy committed:
More generic_skip_line_tt

TheBrit7 committed:
Egypt tree buffs and adjustments

TheBrit7 committed:
Update egypt.txt

TheBrit7 committed:
Egypt loc

BiccyThiccy committed:
Merge branch 'Beta-Edits' into Bugfix-May82025

BiccyThiccy committed:
IRQ - Importing War Effort Additions

BiccyThiccy committed:
IRQ - Importing Most Remaining War Effort Changes

Did a hover test for you, Mariijn pookie :)

Only major missing thing still is the shared Saadabad Pact. @DutchPlanDerLindeSus Since you have the best understanding of your Afghanistan tree, it would be great if you could incorporate it there.

BiccyThiccy committed:
SP shared tree import (still disabled) to reduce errors

BiccyThiccy commented:
@marijn2112 You can ignore the diff check for goe_shared_saadabad_branch.txt as it is just an imported vanilla file

BiccyThiccy committed:
Importing blank RAJ_GOE_alternative_strategy_plan files to cut errors

Brings error log down from 836 -> 744 with all DLC's enabled

BiccyThiccy committed:
Porting (I think?) every GOE Iran change from the War Efforts

Brings down the error log from 744 to 739. Passed the focus-hover test.

BiccyThiccy committed:
Incorporating most if not all missing GOE traits

BiccyThiccy committed:
Replacing the AI_strategy_plan files with 1.16.4 versions for our 1.16.4 tree

BiccyThiccy committed:
IRQ - Importing Maruf al Rusafi

Last commit for today before sleepy

BiccyThiccy committed:
GER - Importing some post-WUW Release Changes

Did a hover test for errors, didn't find anything. But if anyone spots anything plz ping me

BiccyThiccy committed:
Integrating post-WUW German event changes

BiccyThiccy committed:
Porting post-AAT launch Finland changes

marijn211 committed:
Fix broken Swedish mission

marijn211 committed:
Part 2 of fix above

marijn211 commented:
> @marijn2112 You can ignore the diff check for goe_shared_saadabad_branch.txt as it is just an imported vanilla file

I assume you've made changes to it though? Otherwise there's no point in importing it

BiccyThiccy commented:
> > @marijn2112 You can ignore the diff check for goe_shared_saadabad_branch.txt as it is just an imported vanilla file
> 
> I assume you've made changes to it though? Otherwise there's no point in importing it

I'm not gonna lie I'm dumb as a rock, my bad. I think what I meant was that it was an imported vanilla file from 1.16.4. I think I changed it in a later commit anyways so my comment's irrelevant now. Just note that the file isn't intended to function atm, just there to get rid of errors.

BiccyThiccy committed:
Merge branch 'Beta-Edits' into Bugfix-May82025

katsuragiyy committed:
Icons and Fix (?)

BiccyThiccy reviewed (commented):
I did a quick diff-check on the German (and Iraq/Iran/Finland) changes like you asked Marijn, I'm not that great at eyeballing things but I didn't spot anything that looks wrong. The imported vanilla changes I incorporated elsewhere are much simpler thankfully as vanilla didn't shoehorn a huge list of changes like with Germany/India.

I also did hover checks on everything I did on this branch, and haven't found any problems.

BiccyThiccy committed:
Importing vanilla changes to lar_events_l_english.yml

BiccyThiccy committed:
Replacing has_tech = radio_detection with is_special_project_completed

Vanilla seems to do this across the board now, probably just a change we forgot to port.

BiccyThiccy committed:
Fix 2 POR Ideas not using OR = { when they're needed

katsuragiyy committed:
GFX stuffs

katsuragiyy committed:
Bulgaria tt ig?

marijn211 committed:
Ported vanilla UK decolonization bugfix

katsuragiyy committed:
Checked Colombia tree

katsuragiyy committed:
Error fix for Samoa

I did NOT add that

katsuragiyy committed:
Jordan fix and Palestine icon

katsuragiyy committed:
Why are you there???

katsuragiyy committed:
Missing name YUG

katsuragiyy committed:
Korean LAR Advisor

katsuragiyy committed:
Fix blue boy GRE

katsuragiyy committed:
Argentina company names

marijn211 commented on common/characters/KOR.txt:
```diff
@@ -499,6 +499,9 @@ characters={
 				}
 			}
+			visible = {
+				has_dlc = "La Resistance"
```
@CertifiedEnjoyerOfSBG DLC checks like this should go in allowed blocks

marijn211 commented on common/national_focus/samoa.txt:
```diff
@@ -707,8 +707,10 @@ focus_tree = {
-				OVERLORD = {	#they are loosing, best time to leave this sinking boat
-					surrender_progress > 0.2
+				ROOT = {
+					OVERLORD = { #they are loosing, best time to leave this sinking boat
```
This should be equivalent to what it was previously, if it threw an error, this shouldn't fix it

marijn211 committed:
Merge branch 'Beta-Edits' into Bugfix-May82025

marijn211 committed:
Merge branch 'Beta-Edits' into Bugfix-May82025

marijn211 committed:
1 error hopefully fixed

marijn211 reviewed (approved):
![pull request meme](https://github.com/user-attachments/assets/ceebaf5e-57b2-4806-9407-53edb45115e9)

BiccyThiccy committed:
Muh optimization
